docs/quickstart/index.md

```diff
@@ -7,4 +7,98 @@ FileConnector
 --------------
 ```
 libraryDependencies += "dev.zio" %% "zio-connect-file" % "<version>"
-```
\ No newline at end of file
+```
+
+Example
+-------
+
+```scala
+import zio._
+import zio.stream._
+import zio.connect.file._
+
+import java.io.{ByteArrayInputStream, File, IOException}
+import java.nio.charset.StandardCharsets
+import java.nio.file.Path
+
+import scala.language.postfixOps
+import java.nio.file.Files
+import java.nio.file.Paths
+
+object FileConnectorDemo extends ZIOAppDefault {
+  val path = Files.createTempDirectory("zio-connect-file")
```

Comment:
We don't need to use zio-nio. This should also be done with FileConnector.

Reply:
this is using `java.nio` and I guess I wanted to show that you can interact with regular java file apis as well.

Reply:
Let's change it please. 🙏 
We want to show you can do everything without moving away from the FileConnector api